settings.gradle.kts

```diff
@@ -51,7 +53,8 @@ pluginManagement {
 }
 
 plugins {
-    `gradle-enterprise`
+    id("com.gradle.enterprise") version "3.13.2"
+    id("com.gradle.common-custom-user-data-gradle-plugin") version "1.10"
```

Comment:
Usually we keep versions centralized in `gradle.properties` file. Is it possible to use the `pluginManagement` section and the `idv` function to handle this?

Reply:
Done. Since this is a settings plugin, it still needs to be applied here, but the versioning is not handled as you handle other plugins.